database/index.js

```diff
@@ -34,21 +34,15 @@ const daySchema = mongoose.Schema({
 const Day = mongoose.model('Day', daySchema);
 const Event = mongoose.model('Event', eventSchema);
 
-const updateVotes = (timelineId, day, eventId, votes) => {
-  return Day.findAsync({
-    day,
-    timelineId,
-  })
-    .then((results) => {
-      const event = results[0].events.id(eventId)
-      event.votes = votes;
-      return results[0].saveAsync(((err) => {
-        if (err) {
-          console.error(err);
-        }
-      }));
-    });
-};
+const updateVotes = (timelineId, day, eventId, votes) => Day.findAsync({
```

Comment:
Everything looks good, but does findAsync a promise? Seems like it does but i'm unsure